rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_stats.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Check rabbit cluster status
+  shell: "rabbitmqctl cluster_status"
+  register: rabbitmqctl_result
+  delegate_to: "{{ groups['rabbitmq_all'][0] }}"
```

Comment:
We probably *should* do both. Here's why: MaaS is incredibly slow to update. If Murphy's law is in effect, there will be a chance we run into a situation where the alert hasn't triggered and checking manually would also have been useful.